README.md

```diff
@@ -20,27 +20,28 @@ and consistency-availability space. Here are some examples of workloads supporte
 * Replicated state machines
 * Journals of deferred work items
 
-for full documentation, please visit the [logdevice.io](http://logdevice.io/) website
+For full documentation, please visit [logdevice.io](https://logdevice.io/). 
 
 ## Requirements
-LogDevice is supported on Ubuntu 18.04 (Bionic Beaver) only. However, it should
+LogDevice is only supported on Ubuntu 18.04 (Bionic Beaver). However, it should
 be possible to build it on any other Linux distribution without significant
 challenges.
 
-## Building LogDevice
-* [Building](https://facebookincubator.github.io/LogDevice/docs/Installation.html)
+## Quick Start
+* [Run a local cluster in Docker](https://logdevice.io/docs/LocalCluster.html)
 
-## Full documentation
-* [Introduction](https://facebookincubator.github.io/LogDevice/docs/Overview.html)
-* [Cluster Configuration Guide](https://facebookincubator.github.io/LogDevice/docs/Config.html)
-* [LogDevice API](https://facebookincubator.github.io/LogDevice/docs/API_Intro.html)
-* [LogDevice Shell](https://facebookincubator.github.io/LogDevice/docs/LDShell.html)
-* [LDQuery](https://facebookincubator.github.io/LogDevice/docs/LDQuery.html)
```

Comment:
Hmmm, why shouldn't we keep the links to client API, ldshell and ldquery for example? They have corresponding logdevice.io based links.

Reply:
1. The remaining doc links are better starting points, at least in current state.
2. keep it short.
3. one less place to fix if the links change in future.
4. Readers can easily reach those pages once they click on one of the other doc links.

Reply:
Makes sense (Y)